<ui version="4.0" >
 <class>ZipDebugMain</class>
 <widget class="QMainWindow" name="ZipDebugMain" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Drag OpenDocument format OO file to main content...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit" />
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="tabPosition" >
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape" >
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>content.xml</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="XMLTextEdit" name="textEdit" />
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExit" >
    <property name="title" >
     <string>Exit</string>
    </property>
    <addaction name="openzip" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuExit" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="openzip" >
   <property name="text" >
    <string>Open Zip OO</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XMLTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global" >XML_Editor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>ZipDebugMain</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
